src/main/java/com/maxmind/minfraud/request/Email.java

```diff
@@ -74,11 +104,14 @@ public Email build() {
     }
 
     /**
-     * @return The MD5 of the email address used in the transaction.
+     * @return The email address field to use in the transaction. This will be
+     * a valid email address (if you used {@link #address(String)}), an MD5 hash
+     * (if you used {@link #addressMd5(String)}), or null if you did not set an
+     * email address.
      */
     @JsonProperty("address")
-    public String getAddressMd5() {
-        return addressMd5;
+    public String getAddress() {
```

Comment:
This is a breaking change. I think we will want to continue to provide `getAddressMd5()`, at least until we do a major release.

Reply:
That makes sense. Shall we return the MD5 even if we aren't hashing it otherwise?

Reply:
That would probably make sense. I think we could mark the method as deprecated using the `@Deprecated` annotation and documenting the deprecation with the `@deprecated` javadoc tag.